### PR TITLE
Obmin 268 ellipsis text

### PR DESCRIPTION
**Pull request by Voloshkina**
Created ellipsis text , fixed styles back-button and loader

### CODE REVIEW

File: src/components/ellipsis-text/index.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { ITooltip } from 'react-tooltip';
+
+import 'react-tooltip/dist/react-tooltip.css';
+import { EllipsisDiv, ReactTooltip } from './styles';
+
+const delay = 300;
+
+export const EllipsisText = ({
+  id,
+  place = 'bottom-start',
```
Comment (Wolshebnik) [inline src/components/ellipsis-text/index.tsx:10]: sort, pls

File: src/components/ellipsis-text/styles.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  cursor: pointer;
+`;
+
+export const Container = styled.div`
+  display: flex;
+  justify-content: center;
+  max-width: 150px;
+  padding: 10px;
```
Comment (Wolshebnik) [inline src/components/ellipsis-text/styles.ts:38]: padding over max-width

File: src/utils/delay.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+    if (open) {
+      setTimer(null);
+      setPreviewOpen(true);
+      clearTimeout(timer);
+    }
+  }, [open]);
+
+  return [previewOpen, setOpen];
+};
```
Comment (Wolshebnik) [inline src/utils/delay.tsx:25]: If you create new tooltip
Are you sure what you need for this file?

File: src/components/ellipsis-text/EllipsisText.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+export const EllipsisTextWithTooltip = Template.bind({});
+EllipsisTextWithTooltip.args = {
+  children: <span>{text}</span>,
+  id: 'tooltip1',
+};
+
+export const WithAnotherTooltipDirection = Template.bind({});
+WithAnotherTooltipDirection.args = {
+  children: <span>{text}</span>,
```
Comment (Wolshebnik) [inline src/components/ellipsis-text/EllipsisText.stories.tsx:30]: try without span for understanding

File: src/components/ellipsis-text/EllipsisText.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+EllipsisTextWithTooltip.args = {
+  id: 'tooltip1',
+  children: <span>{text}</span>,
+};
+
+export const WithAnotherTooltipDirection = Template.bind({});
+WithAnotherTooltipDirection.args = {
+  id: 'tooltip2',
+  children: text,
+  place: 'right-start',
```
Comment (Wolshebnik) [inline src/components/ellipsis-text/EllipsisText.stories.tsx:32]: sort

**Review by Wolshebnik — APPROVED** [review]

**Review by AkintsevaElizaveta — APPROVED** [review]

**Pull request merged**